_rules/device-motion-user-interface-7677a9.md

```diff
@@ -0,0 +1,355 @@
+---
+id: 7677a9
+name: Device motion can be operated by user interface
+rule_type: atomic
+description: |
+  This rule checks that functionality that can be operated by device motion can also be operated by user interface components.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] that has [functionality][] that can be operated by [device motion][].
+
+## Expectation
+
+For the test target, [device motion][] can also be operated by [user interface components][] and they should be [visible][] and [included in the accessibility tree][] with an [accessible name][] that is not empty ("").
+
+## Assumptions
+
+- This test assumes that the motion to operate [functionality][] is not used through an [accessibility supported][] interface
+- This test assumes that motion is not [essential][] for the [functionality][]
+
+## Accessibility Support
+
+- Device [orientation events][] may include sensitive data. Implementations must fire events only on [secure browsing contexts][]
+
+## Background
+
+- [Understanding Success Criterion 2.5.4: Motion Actuation](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html)
+- [G213: Provide conventional controls and an application setting for motion activated input](https://www.w3.org/WAI/WCAG21/Techniques/general/G213.html)
+- [DeviceOrientation Event Specification](https://www.w3.org/TR/orientation-event/)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The [HTML document][] has [functionality][] that can be operated by [device motion][] and can also be operated by [user interface components][] and they are [visible][] and [included in the accessibility tree][] with an [accessible name][] that is not empty ("").
+
+```html
+<html>
+  <head>
+    <title>Passed Example 1</title>
+    <script>
+      function activateSlider() {
+        const slider = document.getElementById('motionSlider');
+        const output = document.getElementById('output');
+
+        function increaseSlider() {
+          slider.value++;
+          output.innerHTML = slider.value;
+        }
+
+        function decreaseSlider() {
+          slider.value--;
+          output.innerHTML = slider.value;
+        }
+
+        function handleOrientation(event) {
+          if (event.gamma > 20) {
+            slider.value++;
```

Comment:
```suggestion
            increaseSlider();
```
Even if that repeat the `output.innerHTML` change, this make the intend clearer and the fact that both clicking and moving do the same thing immediately obvious.